### ISSUE TITLE
Allow users to scroll using the browser scrollbar

### ISSUE THREAD

**Issue by tomwarner13** [appbaseio/es-queryComposer#81]
When I run a query, a modal opens with the results. For some reason, when this modal is open, the scrollbar functionality of the browser is partially disabled. I can still scroll through the results using the mouse wheel, or page up/down, but I am unable to move the browser's scrollbar by clicking on it. This is mildly annoying with a small result set since it breaks expected behavior, and incredibly annoying on a large result set since if I say, want to scroll back up to the top from being near the bottom it takes ages. Also there's no graphical way to close the modal without scrolling all the way to the top to click that x -- ESC works but that wasn't obvious to me and may not be obvious to others.

So in conclusion, it would be cool if the scrollbar could work consistently with the standard behavior of scrollbars everywhere. Thanks :)
